bin/ch/ch.cpp

```diff
@@ -583,7 +583,11 @@ int main(int argc, char** argv)
         Helpers::NarrowStringToWideDynamic(argv[i], &args[i]);
     }
 
-    int ret = wmain(argc, args);
+    // Call wmain with a copy of args, as HostConfigFlags may change argv
+    char16** argsCopy = new char16*[argc];
+    memcpy(argsCopy, args, sizeof(args[0]) * argc);
```

Comment:
I dont understand- he's not converting from char to char16, he's simply copying from the char16 array called args to the char16 array called newArgs?

---
In reply to: [63589023](https://github.com/Microsoft/ChakraCore/pull/997#discussion_r63589023) [](ancestors = 63589023)